package.json

```diff
@@ -46,21 +47,26 @@
     "@types/ncp": "^2.0.1",
     "@types/nock": "^9.1.3",
     "@types/node": "^10.1.0",
+    "@types/node-fetch": "^2.1.4",
     "@types/pify": "^3.0.2",
     "@types/tmp": "0.0.33",
+    "@types/uuid": "^3.4.4",
     "assert-rejects": "^1.0.0",
+    "await-spawn": "^2.1.1",
```

Comment:
Use [`execa`](https://npm.im/execa) instead as it is more popular and we already use it elsewhere.

Reply:
👍 Let's use https://github.com/googleapis/gcp-metadata/issues/41 to track it.